### PR TITLE
fix: Kafka sensor url. Fixes #1006

### LINKED ISSUE #1006: Kafka Trigger: cannot accept multiple brokers
**Describe the bug**
The same issue was present in the `EventSource` and fixed in #894 . However, the same problem appears in the `Sensor` code (https://github.com/argoproj/argo-events/blob/master/sensors/triggers/kafka/kafka.go#L89) 

**To Reproduce**
1. Create an `EventSource` to receive a message
2. Create a `Source` with the `KafkaTrigger`
3. Set under the `url` a string with multiple kafka brokers url separated by a comma
4. Send a message
5. See error in the `sensor` pod

**Expected behavior**
The expected behaviour should be that you can define multiple brokers URL in the `url` paramater separated by a comma

**Environment (please complete the following information):**
 - Kubernetes: v1.18.1
 - Argo: v2.13.2
 - Argo Events: v1.2.0

**Additional context**
The error you get in the `sensor` pod is the following:

```
{"level":"error","ts":1609935890.4802158,"logger":"argo-events.sensor","caller":"sensors/trigger.go:80","msg":"failed to invoke the trigger","sensorName":"vod-ratings-sensor","trigger":"kafka-ratings","error":"kafka: client has run out of available brokers to talk to (Is your cluster reachable?)","stacktrace":"github.com/argoproj/argo-events/sensors.(*SensorContext).GetTrigger\n\t/home/runner/work/argo-events/argo-events/sensors/trigger.go:80\ngithub.com/argoproj/argo-events/sensors.(*SensorContext).triggerActions\n\t/home/runner/work/argo-events/argo-events/sensors/listener.go:234\ngithub.com/argoproj/argo-events/sensors.(*SensorContext).ListenEvents.func1.3\n\t/home/runner/work/argo-events/argo-events/sensors/listener.go:148"}
```

---

**Message from the maintainers**:

If you wish to see this enhancement implemented please add a 👍 reaction to this issue! We often sort issues this way to know what to prioritize.

### PR DESCRIPTION
**Pull request by spaghettifunk**
This PR solves #1006. It is now possible to submit a string with the kafka brokers urls separated by a comma

### CODE REVIEW

**Comment by spaghettifunk** [comment]
The command `make codegen` fails at some point with the following error:

```
+++++++++++++++++++++++++
++++ GENERATING CRDS ++++
+++++++++++++++++++++++++
Patching manifests/base/crds/argoproj.io_eventbus.yaml
Patching manifests/base/crds/argoproj.io_eventsources.yaml
Patching manifests/base/crds/argoproj.io_sensors.yaml
kustomize build manifests/cluster-install > manifests/install.yaml
kustomize build manifests/namespace-install > manifests/namespace-install.yaml
panic: interface conversion: interface {} is string, not map[string]interface {}

goroutine 1 [running]:
sigs.k8s.io/kustomize/api/resmap.getNamespacesForRoleBinding(0xc0005ec0a0, 0x19)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/api@v0.7.0/resmap/reswrangler.go:417 +0x352
sigs.k8s.io/kustomize/api/resmap.(*resWrangler).SubsetThatCouldBeReferencedByResource(0xc00011b2e0, 0xc0005ec0a0, 0xb, 0xc002bfe401)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/api@v0.7.0/resmap/reswrangler.go:384 +0xdb
sigs.k8s.io/kustomize/api/internal/accumulator.(*nameReferenceTransformer).Transform(0xc0002fe200, 0x555a060, 0xc00011b2e0, 0xc00011b2e0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/api@v0.7.0/internal/accumulator/namereferencetransformer.go:85 +0x531
sigs.k8s.io/kustomize/api/internal/accumulator.(*ResAccumulator).Transform(...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/api@v0.7.0/internal/accumulator/resaccumulator.go:138
sigs.k8s.io/kustomize/api/internal/accumulator.(*ResAccumulator).FixBackReferences(0xc00011b2a0, 0xc00011b2a0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/api@v0.7.0/internal/accumulator/resaccumulator.go:164 +0xbe
sigs.k8s.io/kustomize/api/internal/target.(*KustTarget).makeCustomizedResMap(0xc000856780, 0x0, 0x0, 0x5552100, 0x5fb6700)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/api@v0.7.0/internal/target/kusttarget.go:125 +0x7d
sigs.k8s.io/kustomize/api/internal/target.(*KustTarget).MakeCustomizedResMap(...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/api@v0.7.0/internal/target/kusttarget.go:106
sigs.k8s.io/kustomize/api/krusty.(*Kustomizer).Run(0xc000563d20, 0x7ffeefbffa48, 0x1b, 0x0, 0x0, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/sigs.k8s.io/kustomize/api@v0.7.0/krusty/kustomizer.go:77 +0x29b
sigs.k8s.io/kustomize/kustomize/v3/internal/commands/build.(*Options).RunBuild(0xc000842310, 0x54fcfa0, 0xc000010018, 0x1, 0x0)
	/private/tmp/kustomize-20201210-91507-flrdla/kustomize/internal/commands/build/build.go:148 +0xb5
sigs.k8s.io/kustomize/kustomize/v3/internal/commands/build.NewCmdBuild.func1(0xc0002b1b80, 0xc0005930e0, 0x1, 0x1, 0x0, 0x0)
	/private/tmp/kustomize-20201210-91507-flrdla/kustomize/internal/commands/build/build.go:70 +0xad
github.com/spf13/cobra.(*Command).execute(0xc0002b1b80, 0xc0005930b0, 0x1, 0x1, 0xc0002b1b80, 0xc0005930b0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0002b1600, 0x0, 0xffffffff, 0xc0000a6058)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	/private/tmp/kustomize-20201210-91507-flrdla/kustomize/main.go:19 +0x2a
make[1]: *** [manifests] Error 2
make: *** [codegen] Error 2
```

I'm not sure how to solve it

File: manifests/namespace-install.yaml [hunk 1]
```diff
@@ -1,291 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (whynowy) [inline manifests/namespace-install.yaml:1]: Why is this file gone?

**Comment by whynowy** [comment]
>  strin

What is your `kustomize` version?

File: manifests/namespace-install.yaml (hunk 1 above)
Comment (spaghettifunk) [reply manifests/namespace-install.yaml:1]: I honestly just ran `make codegen` and the result contained this change. I'm not sure why it is that.

**Comment by spaghettifunk** [comment]
My `kustomize` version is 

```
{Version:kustomize/v3.9.0 GitCommit:826b5d9792fb67c4d8f8cd59747698ebf0b22720 BuildDate:2020-12-10T22:29:12+00:00 GoOs:darwin GoArch:amd64}
```

File: manifests/namespace-install.yaml (hunk 1 above)
Comment (whynowy) [reply manifests/namespace-install.yaml:1]: All right, can you just restore this file, and then all the builds should be in good shape. I'll figure out what's wrong there.

BTW, thanks for raising this PR!

Comment (spaghettifunk) [reply manifests/namespace-install.yaml:1]: Done 😃

Comment (whynowy) [reply manifests/namespace-install.yaml:1]: `apiVersion: apiextensions.k8s.io/v1beta1` (line 1) is missing.

Comment (spaghettifunk) [reply manifests/namespace-install.yaml:1]: sorry about that! It's getting late in the evening and I'm getting tired 😄  now it should be fine

**Review by whynowy — APPROVED** [review]
LGTM, thanks!

**Pull request merged**